meta-xt-domd-gen3/recipes-tools/lisot/lisot.bb

```diff
@@ -0,0 +1,24 @@
+DESCRIPTION = "Linux socket test tool"
+SECTION = "extras"
+LICENSE = "MIT"
+PR = "r0"
+
+S = "${WORKDIR}/git"
+
+SRC_URI = " \
+    git://github.com/dterletskiy/lisot.git;protocol=https;branch=main \
+"
+
+LIC_FILES_CHKSUM = "file://LICENSE;md5=70f43a6798ce05aa97a93b0392832ee9"
+
+SRCREV = "${AUTOREV}"
+
+DEPENDS = "git-native"
```

Comment:
Do you really need this?

Reply:
I don't have any insight regarding the rationale for adding this. I guess this maybe be related to cmake somehow. Quick build test w/o adding it have passed. Shall I remove it then?

Reply:
Will remove for now